src/Common/src/Interop/Unix/System.Net.Security.Native/SecuritySafeHandles.cs

```diff
@@ -75,6 +130,12 @@ public void SetGssContext(SafeGssContextHandle context)
             _context = context;
         }
 
+        public void SetAuthenticationPackage(int isNtlm)
```

Comment:
I know I commented on this before, but it still seems to me like this should be a bool, and then if the caller has an int, they should be responsible for translating that int to be a bool.